### DISCUSSION TITLE
Pesquisa Python no Brasil 2024

### DISCUSSION THREAD

**Discussion by lskbr** [apyb/comunidade#125] [Geral]
Pessoal, desde 2022 eu tenho organizado uma pesquisa chamada Python no Brasil.
Desde o início, foi por interesse pessoal e principalmente para tentar entender o perfil dos usuários de Python no Brasil (ferramentas, nível de estudo, salário, etc). Ano passado a participação foi bem menor que em 2022.

Seguindo o conselho de várias pessoas dos grupos no Telegram, eu removi a necessidade de se criar um usuário para responder a pesquisa, a deste ano vai ser anônima, o que acredito incentivar a participação de mais pessoas.

Uma pergunta que apareceu hoje é se a Apyb teria como ajudar na divulgação da pesquisa, talvez até mesmo durante o evento da Python Brasil. Nós poderíamos ajustar as datas para coincidir com o evento em si.

Se alguém quiser continuar a trocar ideias sobre este assunto é só me contactar. As questões estão abertas, ou seja, ainda dá para colocar outras perguntas.

**Comment by villares** [comment]
Acredito que essa pesquisa, e os dados que ela pode produzir, são muito úteis para a nossa comunidade, e deveria ser apoiada! Cabe contribuição de muita gente nesse trabalho e os resultados vão ser muito interessantes.

**Reply by lskbr** [reply]
Resultados de 2023 + dados abertos para download: https://blog.nilo.pro.br/posts/2023-05-08-python-no-brasil-2023/

**Comment by tiidadavena** [comment]
> Pessoal, desde 2022 eu tenho organizado uma pesquisa chamada Python no Brasil. Desde o início, foi por interesse pessoal e principalmente para tentar entender o perfil dos usuários de Python no Brasil (ferramentas, nível de estudo, salário, etc). Ano passado a participação foi bem menor que em 2022.
> 
> Seguindo o conselho de várias pessoas dos grupos no Telegram, eu removi a necessidade de se criar um usuário para responder a pesquisa, a deste ano vai ser anônima, o que acredito incentivar a participação de mais pessoas.
> 
> Uma pergunta que apareceu hoje é se a Apyb teria como ajudar na divulgação da pesquisa, talvez até mesmo durante o evento da Python Brasil. Nós poderíamos ajustar as datas para coincidir com o evento em si.
> 
> Se alguém quiser continuar a trocar ideias sobre este assunto é só me contactar. As questões estão abertas, ou seja, ainda dá para colocar outras perguntas.

Opa, a @apyb/diretoria tem total interesse em apoiar a divulgação! Participamos do apoio da divulgação ano passado e também temos interesse em apoiar a disseminação da iniciativa.

Para a divulgação, podes mandar links, iamgens e textos de divulgação para o email `comunicacao@python.org.br`. Outros pontos essenciais para ajudar na divulgação são a data final para participar da pesquisa, email para contato e critérios de participação.

**Reply by lskbr** [reply]
Muito bom saber disso, ainda não preparei o material deste ano, mas já tenho algumas coisas prontas desde o ano passado. Vou ver se consigo ajuda de um designer para deixar mais bonito também. Já anotei o email. Quanto as datas, estou pensando em algo como entre 1 e 31 de outubro, ai coincide com a Python Brasil, mas dá tempo antes e depois para ter mais participação.

**Comment by gabubellon** [comment]
Acho que essa é uma boa threat pra falar sobre esse meu projeto com teia de aranhas o https://github.com/pybropendata
Onde a ideia é centralizar a distribuição das visualizações dos dados das comunidades python e também ajudar quem precisa colocar alguma visualização no ar

To tentando tirar as teias de aranhas dele e talvez @lskbr podemos ver de pegar as pesquisas anteriores atuais de trazer visualizações de dados pra ca 
Por enquanto só temos algo bem embrionário de algumas python Brasil

**Reply by rougeth** [reply]
Seria bom também a publicação dos dados no repositório https://github.com/pythonbrasil/dados/. 

@apyb/diretoria talvez faça mais sentido esse repositório ficar sob a org da APyB, com isso poderíamos expandir para outros eventos e iniciativas além da Python Brasil.

**Reply by lskbr** [reply]
As visualizações de 2022 foram feitas no Google QQ coisa, mas não é muito estável, pois eles mudam vez ou outra e não tive tempo de atualizar. Mas achei a ideia excelente. 2022 e 2023 já estão disponíveis no site, no fim do post tem o link pro CSV. Passo aqui em outra mensagem.

**Reply by lskbr** [reply]
Dados de 2022:
https://python.nilo.pro.br/cargo/PythonNoBrasil2022/PythonNoBrasilSemuBruto.xlsx
https://python.nilo.pro.br/cargo/PythonNoBrasil2022/PythonNoBrasilNormalizado.xlsx
Google Data Studio: https://datastudio.google.com/reporting/f87f193c-1881-4650-b9dc-5662887e101a
Não gosto de usar nada do Google pq tudo quebra ou é descontinuado :-( Mas tinha.

post: https://blog.nilo.pro.br/posts/2022-04-03-python-no-brasil-2022/

Dados de 2023:
https://static.nilo.pro.br/pythonnobrasil/2023/PythonNoBrasil_resultados_anonimizados_2023.zip
post: https://blog.nilo.pro.br/posts/2023-05-08-python-no-brasil-2023/#resultados-2023

**Comment by lskbr** [comment]
E para este ano, estou  ainda revendo as perguntas/opções. Fiquem a vontade para sugerir novas questões ou opções. Podemos também retirar algumas questões. Eu estou fazendo o possível para não passar de 50 (máximo, pois algumas não são feitas em função das respostas anteriores).

**Comment by lskbr** [comment]
Opa, ontem eu abri o site para 2024: https://pesquisa.logikraft.be/#/
Esta versão já é anônima, ou seja, não pede a criação de usuário para responder.

Eu previ deixar em aberto até 30 de setembro, mas agora vi que a Python Brasil é só em Outubro. Eu posso alongar o período sem problema, até uma semana após a Python Brasil. Ou vocês acham legal já ter os resultados durante a conferência?

Lembrando que eu preciso de ajuda para divulgar o link em si, minhas redes sociais tem um alcance muito pequeno.

**Reply by tiidadavena** [reply]
Olá, Nilo!

O engajamento e o alcance com as contas da APyB também aumentam bastante em outubro. Acho bom estender o prazo para até 24/10, um pouco depois da Python Brasil.

A sua pesquisa conta com algum material gráfico, tipo arte para redes sociais? Se sim, podes anexar aqui porque, em breve, devemos postar iniciativas da comunidade .

**Comment by lskbr** [comment]
Olá, já está aberta até 27/10!
Eu tenho apenas posts e alguns gráficos que saem dos resultados. Vou anexar exemplos:

![mapabrasil_28_09](https://github.com/user-attachments/assets/b9d9865a-2a48-401d-9ea7-8b6da8162d53)
![pythonnobrasil2024_28_09](https://github.com/user-attachments/assets/510fc159-b2d5-4d0a-bcee-1aa3ea43d567)
**Pesquisa Python no Brasil 2024**

Que estado paga melhor seus desenvolvedores Python?

Ferramentas, estudos, trabalho, salário, de tudo um pouco. 

https://pesquisa.logikraft.be 

✅ Anônima, não precisa criar conta 
✅ Confiamos em você 
✅ Resultados abertos
✅ Sem fins comerciais

Não esqueça de compartilhar o link em suas redes sociais.

Em anexo, status de 20/09/24.

Não esqueça de participar e de dar aquela ajuda na divulgação dos links.
